spec/hatchet/absolute_path_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../spec_helper'
+
+#Test that all paths set by the buildpack are absolute instead of relative
+describe "Buildpack paths" do
+  it "are absolute" do
+    buildpacks = [
+      ENV["HATCHET_BUILDPACK_BASE"],
```

Comment:
This needs to be `:default` otherwise it's not testing the current branch, it's always testing main/master.